### PR TITLE
Issue 296 - add position priority for directory tiles

### PR DESCRIPTION
**Pull request by ncloudioj**
This implements #296 , which allows the user to set the position priority level for directory tiles.

### CODE REVIEW

File: front_end/src/components/Tiles/TileForm/TileForm.js [hunk 1]
```diff
@@ -188,6 +188,14 @@ export default class TileForm extends Component {
                     <option value="sponsored">Sponsored</option>
                   </select>
                 </div>
+                <div className="form-group">
+                  <label htmlFor="TilePosition">Position Priority</label>
+                  <select className="form-control" onChange={this.handleChange} id="TilePosition" name="position_priority" ref="position_priority" defaultValue={data.position_priority} >
+                    <option value="medium">medium</option>
+                    <option value="low">low</option>
+                    <option value="high">high</option>
```
Comment (rlr) [inline front_end/src/components/Tiles/TileForm/TileForm.js]: should these be in a more logical order? low to high or high to low? the default can still be medium.

**Comment by rlr** [comment]
looks good! just had that one nit due to my OCD and wanting things in order. But I could be convinced otherwise

**Comment by ncloudioj** [comment]
Fixed! Hope that alleviates your OCD ;)

**Comment by rlr** [comment]
ship it :ship: r+

**Pull request closed without merging**